### PR TITLE
server.js is modified and initialSchema.js is add for migrating the o…

### PR DESCRIPTION
**Pull request by sentayhuJo**
…rganisation data to mongodb

### CODE REVIEW

**Comment by claudiamatosa** [comment]
@sentayhuJo CircleCI is failing the [code style](https://circleci.com/gh/Code-Your-Future/final-project-1/16) checks. In order to replicate this in your terminal for easier fixing, you can run `yarn lint`.

In addition, to get Sublime or Atom to show you linting errors when you are developing, you can install a plugin. For Atom, it's `linter-eslint`, for Sublime text it's Sublime Linter 3.

**Comment by sentayhuJo** [comment]
@claudiamatosa , thanks

File: server/migrateData.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import path from 'path';
+import fs from 'fs';
+import OrganisationSchema from './initialSchema';
+
+const importedFile = path.resolve(__dirname, '../data', 'organisation.json');
+const data = fs.readFileSync(importedFile);
+const allOrganisation = new OrganisationSchema(JSON.parse(data));
+module.exports = allOrganisation;
```
Comment (Baby007mix) [inline server/migrateData.js:8]: well executed server-side script- Lays a good foundation to fetch the data.

Comment (sentayhuJo) [reply server/migrateData.js:8]: @Baby007mix , thanks for your comment.

**Review by Baby007mix — APPROVED** [review]
Well done!

**Pull request merged**